tests/src/mesh/test-cartesian-grid.cpp

```diff
@@ -0,0 +1,124 @@
+/*
```

Comment:
This file is duplicated, is not it?

Reply:
Yes, it seems I forgot to delete it after the test folders were refactored.